src/main.rs

```diff
@@ -1255,19 +1263,33 @@ fn hid_recorder() -> Result<()> {
         data[0..4].copy_from_slice(&bytes_hid_id);
     }
 
-    let skel = open_skel.load().unwrap();
+    Ok(Some(open_skel.load().unwrap()))
```

Comment:
can this fail? Do we need to drop the `unwrap()`?
```rs
Ok(Some(open_skel.load()?))
```

Reply:
it can fail on some occasions, but if it fails, we should bail out (now that we got proper `--bpf` argument handling)